src/app/core/services/search-registration/search-registration.service.ts

```diff
@@ -189,6 +189,14 @@ export class SearchRegistrationService {
     );
   }
 
+  // search my registrations with paging
+  searchMyRegistrations(searchCriteria$: Observable<SearchCriteria>): PagedSearchResult<RegistrationViewModel> {
+    return new PagedSearchResult<RegistrationViewModel>(
+      searchCriteria$,
+      this.searchService.SearchPostSearchMyRegistrations.bind(this.searchService),
+      (searchCriteria) => this.searchService.SearchCount(searchCriteria).pipe(map((result) => result.TotalMatches))
```

Comment:
Dette gir deg nok feil tall, for her har du ikke filter på observatøren. Jeg tror du enten finne en måte å filtrere på observatør på eller implementere en funksjon i API'et for å hente antall av "mine observasjoner".